runbooks/english/05-create-publish-module.md

```diff
@@ -122,3 +122,5 @@ jobs:
 ```
 
 Build the execution step of Renovate scan based on your `RenovateScan` function.
+
+Now, we will expore the Dagger shell, to do same but without any code: [Introducing Dagger Shell](./06-use-interactive-dagger-shell.md)
```

Comment:
As the 5 is like a exploratory (a deeper exercice), may be we can put dagger shell at 5 & 6 in the codelab and put this one as 7. WDYT?